src/elements/content-sidebar/activity-feed/task-new/TaskStatus.js

```diff
@@ -13,32 +12,37 @@ import messages from './messages';
 import type { TaskStatus } from '../../../../common/types/tasks';
 
 import './TaskStatus.scss';
+// $FlowFixMe LabelPill is in typescript
```

Comment:
The flow warnings from this should not prevent the checks from passing. Finding and removing all of these later may be tedious.

Reply:
Removed.

Reply:
@jstoffan looks like lint test fails for these LabelPill imports without the $FlowFixMe...